index.rst

```diff
@@ -10,20 +10,849 @@
+
+Where special training is required is with regards to use of the Catcher, and the development of custom on-the-fly jobs, generation of artifacts, and alerts to the user.
+Because the development of the Catcher framework is in its infancy, a formal training package cannot yet be developed.
+However, upon completion, or at least the implementation of an alpha version, a bootcamp, or series of bootcamps, will be necessary that explains the following items:
+
+   - How to create a trigger for a Catcher job based on the evaluation of a boolean condition (e.g. measured value exceeds a threshold)
+   - The multiple scenarios in which an analysis job can be written and executed
+   - The multiple types of artifacts that can be generated, ranging from a single scalar, to complex data objects, to a png file.
+   - How to archive the artifact
+   - How to display an artifact, including how to deploy a Bokeh App that utilizes the aforementioned complex data object
+   - How to alert a user, specifically an operator, that an artifact is available for viewing (with a level of urgency attached)
+
+
+Useful trainings, but arguably out of FAFF scope, include: preparing for an observation, writing SAL scripts, and operating the observatory via LOVE.
+Also out of scope, but useful to commissioning personnel, are the writing of modules and/or pieces of code that can be added to the Science Pipelines to support FAFF needs.
+For example, using the analysis_tools package to create science performance metrics and diagnostic plots, or adding additional data quality statistics to SFP.
+
+
+.. _Deliverable 9:
+
+Deliverable 9: Task Prioritization
+----------------------------------
+
+.. note::
+
+   The deliverable description from the `charge`_ has been directly copied here to ease readability.
+
+  9. A prioritized list of tasks to build-out the new functionalities with recommended end-dates.
+
+     Where possible, these dates shall correspond to integration milestones.
+
+
+Because much of the work is highly parallelizable, this report groups tasks into prioritization tiers; tasks are not ranked individually within a given tier.
+These tasks are focused on developing high-level architectures.
+The list does not include all the specific displays and/or tools that are required for each individual system (e.g. the AOS GUIs).
+However, a non-exhaustive list of these tools are discussed in the `Recommended Tools`_ section.
+Lastly, the reader should recognize that there is a lot of work to be accomplished that can only be done by small and specific groups of individuals.
+Coordination and management of these tasks will be critical to success of commissioning.
+
+
+Tier 1:
+^^^^^^^
+
+- Complete transition of the Antu servers to the summit.
+  This task is required before many of the Tier 2 tasks can make significant process because the Rapid Analysis payload and supporting framework will run on this cluster.
```

Comment:
Here and in a couple different places it mentions "Rapid Analysis payload" but I think it means "Rapid Analysis workload"... payload is the data generated by a workload..

Reply:
I believe this was addressed in comments above